src/main/resources/stsjorbsmodResources/localization/spa/JorbsMod-Card-Strings.json

```diff
@@ -651,6 +729,15 @@
     "NAME": "Sacudida",
     "DESCRIPTION": "Si un enemigo está Vulnerable, roba !M! cartas. NL Aplica !stsjorbsmod:UrMagic! Vulnerable. NL Agota."
   },
+  "stsjorbsmod:ShrapnelBloom": {
+    "NAME": "Flor de Metralla",
+    "DESCRIPTION": "stsjorbsmod:Expirar. NL Este turno, la próxima carta no-Legendario que juegas se juega !M! vez adicional, después lo stsjorbsmod:Expira.",
+    "UPGRADE_DESCRIPTION": "stsjorbsmod:Exert. NL Este turno, la próxima carta no-Legendario que juegas se juega !M! veces adicionales, después lo stsjorbsmod:Expira."
```

Comment:
Is this supposed to still be "Exert"?

Reply:
Dang! So close I thought I had them all!